### DISCUSSION TITLE
dstnetprefix not working?

### DISCUSSION THREAD

**Discussion by cristianmc** [akvorado/akvorado#542] [Q&A]
![image](https://user-images.githubusercontent.com/10065114/223007681-15aadc66-a8db-4eaa-bcf3-693edc01d185.png)
this filter perfect work, but this not :
![image](https://user-images.githubusercontent.com/10065114/223007795-18f3f3d9-9470-4373-9360-3473ec1337a7.png)

**Comment by vincentbernat** [comment]
It works for me. In the console (F12), could you check the SQL query which is done?

**Reply by cristianmc** [reply]
sql query:
![image](https://user-images.githubusercontent.com/10065114/223246757-cafd6211-18a5-436e-8e80-5179963d381b.png)
with this filter :
![image](https://user-images.githubusercontent.com/10065114/223247003-77067c35-a35b-4e08-a040-e32641a97ee9.png)
im working with 2 exporters now

**Comment by cristianmc** [comment]
update: 
apparently the dstnetprefix filter works, but not with one of the exporters that does not have the external-internal interfaces defined

**Reply by vincentbernat** [reply]
DstNetPrefix matches only when the collected data contains a prefix length. Check that's the case by using it as a dimension instead of a filter.
